race-buffer/src/util.rs

```diff
@@ -0,0 +1,148 @@
+use super::*;
```

Comment:
Could we name this file something more descriptive than `util`? `test_support` maybe? `test_types` ?